modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to cluster roles
+
+As a cluster administrator, you can add unauthenticated users to the following cluster roles in {product-title} by creating a cluster role binding. Unauthenticated users do not have access to non-public cluster roles. This should only be done in specific use cases when necessary.
+
+You can add unauthenticated users to the following cluster roles:
+
+* `system:scope-impersonation`
+* `system:webhook`
+* `system:oauth-token-deleter`
+* `self-access-reviewer`
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+[NOTE]
+====
+Replace the `<cluster_role>` variable with the cluster role you want to update.
+====
+
+. Create a YAML file named `add-<cluster_role>-unauth.yaml` and add the following content:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+ annotations:
+   rbac.authorization.kubernetes.io/autoupdate: "true"
+ name: <cluster_role>access-unauthenticated
+roleRef:
+ apiGroup: rbac.authorization.k8s.io
+ kind: ClusterRole
+ name: <cluster_role>
+subjects:
+ - apiGroup: rbac.authorization.k8s.io
+   kind: Group
+   name: system:unauthenticated
+----
```

Comment:
The Note is rendering a little... weird:
![Screenshot 2024-06-21 at 1 07 19 PM](https://github.com/openshift/openshift-docs/assets/112414607/ec538143-cc6f-425b-ad24-9d662ea0e91b)


At this point, I would consider three options:

1. Moving the note underneath the yaml
2. Removing the note entirely, I'm not sure but I think the meaning of the input variable is already strongly implied.
3. Using callout labels in the yaml and hope that the user knows to apply the same info when creating the file name for the yaml file.

Reply:
You could also play around with other options like adding a + in between .Procedure and the note, but I understand there's not much time right now

Reply:
I think you're right about it being heavily implied and it's probably overkill.